aopy/analysis/connectivity.py

```diff
@@ -0,0 +1,238 @@
+import numpy as np
+from tqdm.auto import tqdm
+import multiprocessing as mp
+
+from . import base
+from .. import data as aodata
+from .. import precondition
+from . import accllr
+
+def get_acq_ch_near_stimulation_site(stim_site, stim_layout='Opto32', electrode_layout='ECoG244', 
+                                 dist_thr=1, return_idx=False):
+    '''
+    Get acquisition channels near a stimulation site. Warning: this function returns channel
+    numbers, which sometimes are 1-indexed. Set the return_idx flag to True to get the channel
+    indices as well.
+
+    Args:
+        stim_site: int, stimulation site
+        stim_layout: str, layout of stimulation sites
+        electrode_layout: str, layout of electrodes
+        dist_thr: float or tuple, threshold for distance from stimulation site
```

Comment:
What units is dist_thr?

Reply:
Also please explain that this can be either a single value where electrodes closer than the threshold are found, or a minimum distance can be defined with a tuple.

Reply:
added a better explanation. i think its typically mm but there's nothing stopping somebody from making a layout with other units